vmaas_sync/vmaas_sync.py

```diff
@@ -274,8 +280,7 @@ def data_received(self, chunk):
 
     def put(self):
         """Answer PUT request."""
-        sync_cve_md()
-        sync_exploits()
+        asyncio.ensure_future(a_sync_cve_md())
```

Comment:
There should be a protection to ensure the sync is a singleton. Running the API repeatedly might cause some database issues/crashes.